hack/golangci_lint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+source hack/common.sh
+
+# check for golangci-lint in *.go files
+
+check=0
+
+test_golangci() {
+        "${OUTDIR_TOOLS}"/golangci-lint run ./...
+        if [ $? -ne 0 ]; then
+                check=1
+        fi
```

Comment:
I'm sorry I didn't catch this earlier, but you shouldn't need to do this. The function `test_golangci` will automatically return the return code of the last statement it executes. Being that there's only one command in this function, it really doesn't need to be it's own function at all. you can just run the command in the main block. I would also recommend setting `set -e` at the top of the file, since if at any point we hit an error it would make sense to exit the script.